spec/demo_data/fake_student_spec.rb

```diff
@@ -18,10 +18,23 @@
     expect(student.home_language).not_to be_nil
   end
 
+  it "sets student enrollment status to be either 'Active' or 'Transferred'" do
+    expect(student.enrollment_status).to eq('Active').or(eq('Transferred'))
+  end
+
+  it "sets student race attributes" do
+    expect(student.enrollment_status).not_to be_nil
```

Comment:
The comment mentions race attributes, but the spec checks enrollment status -- looks like a simple copy-paste thing, easy fix.